### ISSUE TITLE
Remaining tasks after the completion of the effort to get the refman checked by `doc_grammar`.

### ISSUE THREAD

**Issue by Zimmi48** [coq/coq#16652]
With #16498 being close to done, we are getting near the point where all documented syntax in the reference manual except the SSReflect chapter is automatically checked against the actual grammar of Coq. The purpose of this issue is to collect the remaining tasks to attend after this PR is merged.

- [ ] `specialize` and `contradiction` are postponed to a next PR. Also postponed to a next PR is deciding whether to restructure the Tactics chapter, change the internal section names or even the name of the chapter itself.
- [ ] In the Ltac chapter, there remains two uses of `.. tacv::` (these are unchecked against the Coq grammar) that document things coming from #676. We should clarify what these are and add them to `common.edit_mlg` if relevant.
- [ ] After that, we should decide what to do of `.. tacv::` (use it but make it checked by `doc_grammar` or remove it). Same for `.. cmdv::`.
- [ ] Since there is no short or medium-term plan to apply the same process to the SSReflect chapter (after the failure of #13019 to attract volunteer reviewers), there is no point to keep including the SSReflect grammar in `fullGrammar` and `orderedGrammar` (this just makes the contributing process more complex without bringing the associated benefits). So we should remove the SSReflect `.mlg` files from the sources used by `doc_grammar`.
- [ ] After that, we can consider adding a linter check that the full syntax in `orderedGrammar` is documented somewhere in the reference manual. This means that it will no longer happen that a tactic or command can be added without documentation (except in the SSReflect plugin).
- [ ] All the existing tactics and commands are now (or will then be) listed in the reference manual, but some of them are still marked as `:undocumented:`. We should decide for each of them, whether we want to document them or remove them (sooner rather than later, and ideally for 8.17). This includes the already opened PRs #16561 and #16407. We should eventually limit the use of `:undocumented:` to the `.. exn::` and `.. warn::` cases only.
- [ ] We should update the documentation for contributing to the documentation so that it is up-to-date (this includes `CONTRIBUTING.md`, `doc/sphinx/README.rst` and `doc/tools/docgram/README.md`).
- [ ] We should go through `common.edit_mlg` and decide which changes might be worth "upstreaming" to the `.mlg` files.

Some of these tasks depend on previous tasks, but most of them could be moved forward in parallel.

**Comment by jfehrle** [comment]
A reasonable list.

> Also postponed to a next PR is deciding whether to restructure the Tactics chapter, change the internal section names or even the name of the chapter itself.

We should pick some priorities for the next round of doc improvements in the near future--after we complete this list.

**Comment by jfehrle** [comment]
Was there a reason to omit these tactics from the doc?  I vaguely recall some discussion.

```
typeclasses eauto best_effort {? @nat_or_var }
typeclasses eauto best_effort {? @nat_or_var } with {+ @ident }
```

**Comment by jfehrle** [comment]
Where should this production be documented and what does it mean?  IIUC it is for patterns.

```
term10: [
| "@" pattern_ident LIST1 identref
```
